src/pylast/__init__.py

```diff
@@ -547,7 +547,7 @@ def scrobble(
             title (Required) : The track name.
             timestamp (Required) : The time the track started playing, in UNIX
```

Comment:
Thanks! Whilst we're here, let's change these. Please could you also do the one on line 2385?

```suggestion
            timestamp (Required) : The time the track started playing, in Unix
```

Reply:
@hugovk Done ✅